build/package/scripts/build-python.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -eu
+
+NO_PROXY="${NO_PROXY:-}"
+HTTPS_PROXY="${HTTPS_PROXY:-}"
+
+printf "\nInstall test requirements\n" 
+. /opt/venv/bin/activate
+pip install --upgrade pip
+if [ -z $HTTPS_PROXY ]; then pip install -r tests_requirements.txt; else pip install --proxy $HTTPS_PROXY -r tests_requirements.txt; fi
+pip check
+
+printf "\nExecute linting\n"
+mypy src
+flake8 --max-line-length=120 src
+
+printf "\nExecute testing\n"
+mkdir -p build/test-results/test
+mkdir -p build/test-results/coverage
+PYTHONPATH=src python -m pytest --junitxml=build/test-results/test/report.xml -o junit_family=xunit2 --cov-report term-missing --cov-report xml:build/test-results/coverage/coverage.xml --cov=src -o testpaths=tests
```

Comment:
since we don't use anymore the path pattern `build/test-results/test` we could change this to local path or another specific/shorter one

Reply:
I saw @michaelsauter using it in the ods-build-go task, so I followed the same approach, but happy to change it if needed :)

Reply:
yes we could discuss changing this since we have a new canonical path, which is under `.ods/artifacts`. So each tech can actually place the test report wherever they want, as long as they copy it to the artifacts later.